lib/auth/clt.go

```diff
@@ -1540,10 +1540,34 @@ func (c *Client) GetSignupTokenData(token string) (user string, otpQRCode []byte
 	return tokenData.User, tokenData.QRImg, nil
 }
 
+// GenerateUserCertSet takes the public key in the OpenSSH `authorized_keys` plain
+// text format, signs it using User Certificate Authority signing key and
+// returns the resulting certificates.
+func (c *Client) GenerateUserCertSet(key []byte, user string, ttl time.Duration, compatibility string) (*proto.CertSet, error) {
```

Comment:
because it's a new method, may as well start accepting context as a first parameter instead of using `context.TODO()`